primes.rb

```diff
@@ -1,6 +1,16 @@
 class Primes
   def self.sum_to(limit = 100)
     # TODO - add your prime number solution here...
-    "I'm working on it!"
+    require 'prime'
+
+	total = 1
+
+	(1..limit).each do |i|
+		break if ((total + i) > limit)  
+		total += i if (Prime.prime?(i))
```

Comment:
Hey Dan, I would like to "cite" here the changes I produced in last commit (26827f0), but I can't find a method for it here on github.

Also, I added a comment here below that I would like to delete, but same story.

Any help?

What I want to achieve is display the new primes.rb here as answer to your comment, basically, with some words from me as well.